<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceForm</class>
 <widget class="QDialog" name="DeviceForm">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>310</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>335</width>
    <height>180</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Oculus Setup</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>280</y>
     <width>231</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>341</width>
     <height>261</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="networkTab">
    <attribute name="title">
     <string>Network</string>
    </attribute>
    <widget class="QSlider" name="dhcpEnable">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Enables or disables the IP address assignment via DHCP</string>
     </property>
     <property name="maximum">
      <number>1</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickInterval">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>IP Address:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>DHCP Enable:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Subnet Mask:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ipMask">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>130</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Fixed subnet mask of Oculus</string>
     </property>
     <property name="inputMask">
      <string>999.999.999.999;_</string>
     </property>
     <property name="text">
      <string>999.999.999.999</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ipAddr">
     <property name="geometry">
      <rect>
       <x>169</x>
       <y>80</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Fixed IP address of Oculus</string>
     </property>
     <property name="inputMask">
      <string>999.999.999.999;_</string>
     </property>
     <property name="text">
      <string>999.999.999.999</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QSlider" name="netSpeedLimitControl">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>180</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Slide to alter the maximum network speed</string>
     </property>
     <property name="minimum">
      <number>10</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickInterval">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Speed Limit</string>
     </property>
    </widget>
    <widget class="QLabel" name="netSpeedLimitValue">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>180</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>100</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>180</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Mbit/s</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="advancedTab">
    <attribute name="title">
     <string>Advanced</string>
    </attribute>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Gain Assist Control:</string>
     </property>
    </widget>
    <widget class="QSlider" name="gainAssist">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Enables or disables the gain assistance</string>
     </property>
     <property name="maximum">
      <number>1</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickInterval">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Gamma Control:</string>
     </property>
    </widget>
    <widget class="QSlider" name="gammaControl">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>80</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Slide to adjust the amount of gamma correction</string>
     </property>
     <property name="minimum">
      <number>127</number>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="value">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickInterval">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="gammaValue">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>80</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>1.00</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DeviceForm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>167</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DeviceForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>167</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
